e2e/libs/keywords/volume_keywords.py

```diff
@@ -111,7 +111,18 @@ def delete_replica_on_node(self, volume_name, replica_locality):
             node_name = self.get_node_id_by_replica_locality(volume_name, replica_locality)
 
         logging(f"Deleting volume {volume_name}'s replica on node {node_name}")
-        self.volume.delete_replica(volume_name, node_name)
+        retry_count, retry_interval = get_retry_count_and_interval()
+        for _ in range(retry_count):
+            try:
+                self.volume.delete_replica(volume_name, node_name)
+                return
+            except Exception as e:
+                logging(f"Deleting volume {volume_name}'s replica on node {node_name} failed with error: {e} ... ({_})")
+                time.sleep(retry_interval)
+
+    def delete_unscheduled_replica(self, volume_name):
+        logging(f"Deleting volume {volume_name}'s unscheduled replica")
+        self.volume.delete_unscheduled_replica(volume_name)
```

Comment:
Should we put this in the try too?

Reply:
So far I haven't run into errors with this step, so I'd prefer to keep it simple for now.